React/Profiler/RCTPerfMonitor.m

```diff
@@ -74,11 +77,11 @@ static vm_size_t RCTGetResidentMemorySize(void)
   return info.resident_size;
 }
 
-@class RCTDevMenuItem;
```

Comment:
This shouldn’t be necessary, as the class should exist when `RCTDevMenu.h` is imported.